idx/memory/memory.go

```diff
@@ -552,13 +552,18 @@ func (m *MemoryIdx) resolveIDs(ids TagIDs) []string {
 			continue
 		}
 
-		res[i] = def.NameWithTags()
+		res[i] = idx.Node{
+			Path:        def.NameWithTags(),
+			Leaf:        true,
+			HasChildren: false,
```

Comment:
this seems incorrect. why are we hardcoding these values?
You can see in `Find()` and `find` how it's done. basically pull the `memory.Node` out of `tree.Items` via the path and then : 
```
			idxNode := idx.Node{
				Path:        n.Path,
				Leaf:        n.Leaf(),
				HasChildren: n.HasChildren(),
			}
			if idxNode.Leaf {
				idxNode.Defs = make([]idx.Archive, 0, len(n.Defs))
				for _, id := range n.Defs {
(...)
```